core-plugins/docs/FTP-batchsource.md

```diff
@@ -0,0 +1,40 @@
+# FTP/SFTP Batch Source
```

Comment:
An alternative would be to call it an ``SFTP Batch Source``, where FTP is just one of the other protocols it handles.